spec/factories/registration.rb

```diff
@@ -15,7 +15,8 @@
     registration_exemptions { build_list(:registration_exemption, 3) }
 
     trait :was_renewed do
-      referring_registration { build(:registration) }
+      # referring_registration_id { build(:registration) }
```

Comment:
I know you said something about needing to do this to get the test to pass. Could you add a `TODO: ` comment to that effect here e.g.

```ruby
# TODO: Ideally we'd like the referring registration to actually be a
# registration, but this seems to break the tests. We need to investigate why
# but we are happy this is valid as an interim measure.
# referring_registration_id { build(:registration) }
referring_registration_id { "123" }
```

Reply:
Comment added as requested.